### PR TITLE
Style Footer with Formidable space brand styles

### PR DESCRIPTION
**Pull request by paulathevalley**
Resolves https://github.com/FormidableLabs/formidable-landers/issues/136

This PR: 
- Style `<Footer />` component to match the Formidable space brand 
- Add support for trademark messaging
- In the case where header & footer might want different themes, change the scoped classes to `.formidable-header` and `.formidable-footer`, respectively 
- Add 2 theme options: light and dark, with the ability to override styles 

Would love some review on styles and copying assets, whenever someone has a chance!

screenshot below shows: 
default header
header with its own children
default footer
footer with its own `trademark` prop 
screen shot 2016-08-31 at 2 02 57 pm

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
     "test-travis": "karma start ./karma.conf.js --singleRun=true",
     "babel": "rimraf lib && babel src/ -d lib/",
     "lint": "eslint --ext .js,.jsx src",
-    "prepublish": "npm run babel",
+    "prepublish": "npm run babel && cp -r src/assets/ lib/assets/",
```
Comment (paulathevalley) [inline package.json]: :eyes: Is this right?

Comment (coopy) [inline package.json]: If `src/assets` will always contain _all the assets_ (source of truth), I'd recommend cleaning out `lib/assets` first:

```sh
rm -rf lib/assets/*
```

If `lib/assets` isn't guaranteed to exist, then before any `rm` or `cp`, do:

```sh
mkdir -p lib/assets
```

**Comment by coopy** [comment]
1 comments, all the other changes look good to me! 🚀 after changing the `prepublish` script

**Comment by chrisbolin** [comment]
Hey @paulathevalley! I know I'm late to the party. But I had one thing to add...

It might be helpful to abstract the colors to variables, and probably to a separated shared file. This might come in handy when adding/editing components. For example, `#242121` appears 13 times between the header, footer, and specs. That leads to a lot of find/replace when refactoring. And it might be even harder for colors (e.g. `#fff`) that could have served multiple purposes, like background and text.

Just my 0.02 USD :)

**Pull request merged**